tests/System.Buffers.Primitives.Tests/UsageScenarioTests.cs

```diff
@@ -41,8 +41,6 @@ public void CtorSpanOverByteArrayValidCasesWithPropertiesAndBasicOperationsCheck
             Span<byte> span = new Span<byte>(array);
             Assert.Equal(array.Length, span.Length);
 
-            Assert.NotSame(array, span.ToArray());
```

Comment:
It seems like in 3.0 this behavior changed and span.ToArray returns the same, or the test was wrong.